scalableminds/webknossos
# Map node group context menu to right-click

This PR move the functionality to bring up a context-menu on node groups to a right-click action instead of having to over of the little gear icon. This make is feel more in line with many other webKnossos interactions, that are now triggered through right clicks.

<img width="467" alt="image" src="https://user-images.githubusercontent.com/1105056/152330747-6b2d9115-3251-4faf-a2ea-51bfc8c2b1d1.png">


### Steps to test:
- Create a new node group
- Right click on node group to bring up context menu
- Confirm all the available group related actions from the menu still work

### Issues:
- fixes #5258

------
(Please delete unneeded items, merge only when none are left open)
